### PR TITLE
Add sponsorssection aa2 to master

### PR DESCRIPTION
**Pull request by kleberbaum**
The content of the sponsor text has been adjusted.

**Checklist:**

 
 - [ ] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [ ] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [x] Documentation (non-breaking change which adds documentation)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently the content of the sponsor text is not fitting.

**What is the new behavior (if this is a feature change)?**
- Now the content of the sponsor text is ok.

### CODE REVIEW

File: src/components/organisms/sections/Sponsors/index.jsx [hunk 1]
```diff
@@ -28,11 +28,11 @@ class Sponsors extends React.PureComponent {
...
         <MDBContainer className="py-5 text-center text-dark">
           <h2 className="gidole h1-responsive font-weight-bold">
-            Many companies are already convinced
+            Special thanks to our sponsors
           </h2>
           <p className="lead">
-            Become a <strong>sponsor</strong> and support us in the further
-            development of our <strong>open source</strong> social network.
+            Without the help and support of our <strong>sponsor</strong> the development of
+            our <strong>sponsor</strong> social network SNEK would hardly be possible.
```
Comment (kleberbaum) [inline src/components/organisms/sections/Sponsors/index.jsx]:
```suggestion
            Without the help and support of our <strong>sponsors</strong> the development of
            our <strong>open source</strong> social network SNEK would hardly be possible.
```

**Review by kleberbaum — COMMENTED** [review]
It seems like this text still needs some adjustment.

**Comment by schettn** [comment]
please merge add-sponsorssection-aa2 into implement-1

**Review by schettn — APPROVED** [review]
yep

**Review by pinterid — APPROVED** [review]
LGTM

**Pull request merged**